### ISSUE TITLE
Hope to add download control component

### ISSUE THREAD

**Issue by uni-pai** [fengqiaozhu/joplin_plugin_nlr#6]
When downloading more than a few hundred chapters, the download will always pause itself and you must restart the app to continue downloading. However, after several restarts, the download task is cancelled by itself. Also, the download does not support continue downloading. When you click on the download again, it will start from the beginning, so you will never be able to download the whole novel.
I hope that the download control can be added or continue downloading can be supported.

**Comment by fengqiaozhu** [comment]
You are right, this plugin need a download state management, and I just plan to  spend some time to improve this  plugin.
